src/SIL.XForge.Scripture/Services/SFProjectService.cs

```diff
@@ -361,6 +361,8 @@ public async Task CancelSyncAsync(string curUserId, string projectId)
         await _syncService.CancelSyncAsync(curUserId, projectId);
     }
 
+    public new async Task<SFProject> GetProjectAsync(string projectId) => await base.GetProjectAsync(projectId);
```

Comment:
Maybe just mark the method as public in the base class? Or is there more going on here that I'm missing?